### PR TITLE
Prepare for 0.12.0 release

### PR DESCRIPTION
**Pull request by Yatekii**

### CODE REVIEW

**Review by Dirbaio — APPROVED** [review]
bors r+

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
+## [0.12.0]
+
+### Changed
+
+- Update to probe-rs 0.11.0.
```
Comment (adamgreig) [inline CHANGELOG.md:18]:
```suggestion
- Update to probe-rs 0.12.0.
```

**Review by adamgreig — CHANGES_REQUESTED** [review]
bors cancel

**Review by Dirbaio — APPROVED** [review]
bors r+

**Pull request merged**